build/lib/locFunc.ts

```diff
@@ -24,33 +21,26 @@ export function packageLangpacksStream(): NodeJS.ReadWriteStream {
 		})
 
 	const builtLangpacks = langpackDescriptions.map(langpack => {
-		return fromLocalNormal(langpack.path)
+		return ext.fromLocalNormal(langpack.path)
 			.pipe(rename(p => p.dirname = `langpacks/${langpack.name}/${p.dirname}`));
 	});
 
 	return es.merge(builtLangpacks);
 }
 
-//copied from extensions.
-function fromLocalNormal(extensionPath: string): Stream {
-	const result = es.through();
-
-	const vsce = require('vsce') as typeof import('vsce');
-
-	vsce.listFiles({ cwd: extensionPath, packageManager: vsce.PackageManager.Yarn })
-		.then(fileNames => {
-			const files = fileNames
-				.map(fileName => path.join(extensionPath, fileName))
-				.map(filePath => new File({
-					path: filePath,
-					stat: fs.statSync(filePath),
-					base: extensionPath,
-					contents: fs.createReadStream(filePath) as any
-				}));
-
-			es.readArray(files).pipe(result);
+// Modified packageLocalExtensionsStream but for any ADS extensions including excluded/external ones.
+export function packageSingleADSExtensionStream(name : string): NodeJS.ReadWriteStream {
```

Comment:
This doesn't really seem to have anything to do specifically with ADS extensions - could this just be named `packageSingleExtensionStream` or something to make it clearer?